packages/loader/container-loader/src/containerStorageAdapter.ts

```diff
@@ -72,38 +112,48 @@ export class ContainerStorageAdapter implements IDocumentStorageService {
         scenarioName?: string,
         fetchSource?: FetchSource,
     ): Promise<IVersion[]> {
-        return this.storageGetter().getVersions(versionId, count, scenarioName, fetchSource);
+        return this._storageService.getVersions(versionId, count, scenarioName, fetchSource);
     }
 
     public async uploadSummaryWithContext(summary: ISummaryTree, context: ISummaryContext): Promise<string> {
-        return this.storageGetter().uploadSummaryWithContext(summary, context);
+        return this._storageService.uploadSummaryWithContext(summary, context);
     }
 
     public async downloadSummary(handle: ISummaryHandle): Promise<ISummaryTree> {
-        return this.storageGetter().downloadSummary(handle);
+        return this._storageService.downloadSummary(handle);
     }
 
     public async createBlob(file: ArrayBufferLike): Promise<ICreateBlobResponse> {
-        return this.storageGetter().createBlob(file);
+        return this._storageService.createBlob(file);
     }
 }
 
 /**
  * Storage which only supports createBlob() and readBlob(). This is used with IDetachedBlobStorage to support
  * blobs in detached containers.
  */
-export class BlobOnlyStorage implements IDocumentStorageService {
+class DetachedStorageWrapper implements IDocumentStorageService {
```

Comment:
notCalled() has some strings with the previous name still

Reply:
good catch. i'll just revert the rename, as i'd like the errors to stay consistent

Reply:
It might make sense to just call notCalled() from verifyStorage(), or maybe better would be to throw more explicit errors indicating that the call is not allowed in detached, or it is allowed but no detached blob storage was provided.